tls/x509utils/certpool/system_none.go

```diff
@@ -0,0 +1,9 @@
+//go:build !linux
```

Comment:
`!linux` is not `*_none.go`, but then again having `*_windows.go`, `*_ios.go`, `*_darwin.go` with the same content is dumb. `*_nonlinux.go` is also kind of crazy.

Reply:
my intention was adding extra exclusions (`!foo`) once `system_foo.go` is implemented

Reply:
`system_todo.go` maybe?